osf-cli/README.md

```diff
@@ -0,0 +1,83 @@
+#OSF Command Line Interface 
+This module supplies a command line interface (CLI) for retrieving a registration from an OSF instance and writing it locally into a package which conforms to the Data Conservancy packaging specification.
+
+##How It Works
+The CLI uses functionality in the `osf-packager` module and the OSF Java client in the `osf-client` package to attach to the OSF REST APIs (both v1 and v2) on a running OSF instance specified by a configuration file whose location must be supplied by the user. Additional parameters will need to be supplied as indicated below. The packager code leverages a workflow from the Dataconservancy Package Tool GUI to construct the package once the content from the target registration has been retrieved. The package is then saved to a location specified by the user as a command line option.
+
+
+##Command Line Usage
+The CLI takes a single argument which is a URL pointing to the registration whcih is to be packaged (for example, `http://192.168.99.100:8000/v2/registrations/vacbu/`
+This is preceded by a list of command line options as follows:
+
+```
+-c (-configuration, --configuration)  FILE   : path to the OSF Java client configuration
+-h (-help, --help)                           : print help message
+-m (-metadata, --metadata) FILE              : the path to the metadata properties file for additional bag metadata
+-n (-name, --name) VAL                       : the name for the bag
+-o (-output, --output) FILE                  : path to the directory where the package will be written
+-v (-version, --version)                     : print version information
+```
+The `-c, -m, -n` and `-o` options are required. The OSF Java client must be configured so that it knows which running OSF instance to attach to. Additional metadata is specified in a bag metadata properties file, and is used to describe the bag in accordance with the Data Conservancy BagIt Profile. The package name is used to both name the package and the bag. Finally, the output location will tell the CLI where to write the package.
+
+###OSF Java Client Configuration
+Configuration mut be supplied for both the v1 and v2 endpoints, since both are needed to build the package.  An example is below:
+```
+{
+  "osf": {
+    "v2": {
+      "host": "192.168.99.100",
```

Comment:
I'm wondering if this config should be updated to reflect connecting to the canonical endpoints for the OSF API and Waterbutler.  That way users can (mostly) copy and paste, with the exception of the `authHeader`.